staging/src/k8s.io/client-go/tools/remotecommand/v2.go

```diff
@@ -195,5 +195,8 @@ func (p *streamProtocolV2) stream(conn streamCreator) error {
 type errorDecoderV2 struct{}
 
 func (d *errorDecoderV2) decode(message []byte) error {
-	return fmt.Errorf("error executing remote command: %s", message)
+	if len(message) > 0 {
+		return fmt.Errorf("error executing remote command: %s", message)
+	}
+	return nil
```

Comment:
I thought this is the behavior we want to fail, no?

Reply:
So I was under the impression that we only started explicit success messages in v4, and this is v2 - if I don't add this case, wouldn't that break v2 compatability?